cloudinstall/machinewait.py

```diff
@@ -124,6 +124,7 @@ def update(self):
         # ensure that the button is always focused:
         self.main_pile.focus_position = len(self.main_pile.contents) - 1
 
+    # XXX: Is this utils.async needed?
```

Comment:
I think yes, see how do_install is called elsewhere, like
single.py. do_install() does a ton of stuff, and in order to not block
the UI updates it should be called with the wrapper
do_install_async(). I think we have a non-async version just for
headless mode.